wordlist.txt

```diff
@@ -70,6 +70,7 @@ HistoryQuery
 historyResponse
 HistoryResponse
 HistoryRPC
+hoc
```

Comment:
how can we have ad without hoc!

Reply:
I vote we import a complete Latin dictionary. :D